src/library/baseplaylistfeature.cpp

```diff
@@ -566,9 +558,20 @@ void BasePlaylistFeature::slotExportPlaylist() {
             //default export to M3U if file extension is missing
             if(!file_location.endsWith(".m3u", Qt::CaseInsensitive))
             {
-                qDebug() << "Crate export: No valid file extension specified. Appending .m3u "
+                qDebug() << "Playlist export: No valid file extension specified. Appending .m3u "
                          << "and exporting to M3U.";
                 file_location.append(".m3u");
+                if(QFileInfo(file_location).exists()) {
```

Comment:
spaces after parentheses
```suggestion
                if (QFileInfo(file_location).exists()) {
```
looks like this is an issue with surrounding code too, so please fix that as well